roles/platform/tasks/setup_gcp_env.yml

```diff
@@ -25,6 +25,7 @@
     public_ip: "{{ plat__use_public_ip }}"
     log_location: "gs://{{ plat__gcp_storage_location_logs }}"
     log_identity: "{{ plat__gcp_log_identity_name }}@{{ plat__gcp_project }}.iam.gserviceaccount.com"
+    backups_location: "gs://{{ plat__gcp_storage_location_backups }}"
```

Comment:
This cloudera.cloud.env parameter is called `backup_location`. 

I think it's because we used the change from https://github.com/cloudera-labs/cloudera.cloud/pull/95 instead of https://github.com/cloudera-labs/cloudera.cloud/pull/80.

Reply:
I guess it would make sense changing the rest of the variable names (`infra.gcp.storage.path.backups`, `env.gcp.storage.path.backups`, `common__backups_path`, etc) also to singular for consistency?

Reply:
Yes that would be good. Although I see that log(s) has the same inconsistencies!